packages/carbon-graphs/src/js/controls/Graph/helpers/helpers.js

```diff
@@ -511,6 +511,16 @@ const updateXAxisDomain = (config, input = {}) => {
     // are converted to epoc.
   }
 
+  // if the x-axis type is timeseries then convert to epoc date so it is a number for calculations
+  if (config.axis.x.type === AXIS_TYPE.TIME_SERIES) {
```

Comment:
Nit: Appears to be the same as the `if` block above, was this duplicated unintentionally?

Reply:
Oh, looks like it got duplicated during rebasing. Nice Catch!
Removed here: 0d1a99ea9d665e611651f696c5f773970f04ce0c